src/components/Project/index.js

```diff
@@ -300,22 +300,22 @@ class Project extends React.Component {
         </Grid>
         <Switch>
           <Route exact path='/projects/:slug/:environment/features' render={(props) => (
-            <ProjectFeatures history={history} socket={socket} {...props} />
+            <ProjectFeatures history={history} socket={socket} {...props} environment={this.props.environment}/>
```

Comment:
since we're passing in `...props`, is there anyway environment can be extracted from within there?